tools/convert_mapped_tags.rb

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+require File.expand_path('../config/environment', __dir__)
+
+read_only = (ARGV[0] != 'WRITE')
+
+puts
+puts
+
+if read_only
+  puts "READ ONLY MODE"
+else
+  puts "WRITE MODE"
+end
+
+puts
+
+condition_for_mapped_tags = ContainerLabelTagMapping::TAG_PREFIXES.map { "tags.name LIKE ?" }.join(' OR ')
+tag_values = ContainerLabelTagMapping::TAG_PREFIXES.map { |x| "#{x}%:%" }
+
```

Comment:
It may be good to do this in a transaction and roll back if any of the saves fail